assets/quick-add.css

```diff
@@ -44,9 +45,10 @@
 
 @media screen and (min-width: 750px) {
   .quick-add-modal__content {
+    --modal-height-offset: 20rem;
```

Comment:
Hmm is it different contexts in that way though? In either case it represents the offset distance between a top/bottom edge of the modal and the edge of the viewport. It's used in the max-height calculation for desktop to effectively set a bottom offset. It doesn't use `bottom` or `margin-bottom` for this on desktop I believe because of the specific way we wanted the modal to scale/overflow.

I was however using it to represent the top or bottom as well as the combined offset of top/bottom, which is indeed odd. So now `modal-height-offset` will only represent the offset from either, and should be multiplied by 2 when the total space isn't needed. I think this makes more sense. lmk if I'm looking at this wrong though.